admin_guide/topics/creating-golden-images-cluster-auto-scaler.adoc

```diff
@@ -0,0 +1,112 @@
+// Module included in the following assemblies:
+//
+// * admin_guide/cluster-autoscaler.adoc
+
+[id='creating-golden-image-cluster-auto-scaler-{context}']
+= Creating prime image (aka golden images)
+
+[NOTE]
+====
+If you already have a pre-existing primed image you can skip this
+section.
+====
+
+You can use Ansible playbooks to automatically create and seal a
+primed image that is suitable for use by a LC and its ASG. All the
+attibutes in the following steps are from your existing AWS cluster.
+
+.Procedure
+
+. Create an Ansible *_inventory_* file
```

Comment:
This is really your desktop (or laptop). It is a local file because the invocation of `ansible-playbook` will reference this file and it will also need a path to the `build_ami.yml` playbook.